examples/hello-world/hello-pt-resnet/src/simple_network.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Should we rename the file to be resnet.py?

Reply:
renamed